features/min_ssh_bootstrap_xmlrpc.feature

```diff
@@ -0,0 +1,50 @@
+Feature: register a salt-ssh system via XMLRPC API bootstrap procedure
+
+  Scenario: Setup XMLRPC Bootstrap: Delete ssh-minion system profile before XMLRPC bootstrap test
+    Given I am on the Systems overview page of this "ssh-minion"
+    When I follow "Delete System"
+    And I should see a "Confirm System Profile Deletion" text
+    And I click on "Delete Profile"
+    Then I should see a "has been deleted" text
+    And I cleanup minion: "ssh-minion"
+
+  Scenario: bootstrap a sles ssh-minion via XMLRPC API (it will be deleted after)
+    Given I am logged in via XML-RPC/system as user "admin" and password "admin"
+    When I call system.bootstrap() on host "ssh-minion", SSH user name "root", password "linux" and saltSSH "enabled", a new system should be bootstraped.
+    And I logout from XML-RPC/system.
+
+  Scenario: check new XMLRPC bootstrapped salt-ssh in System Overview page
+     Given I am authorized
+     And I navigate to "rhn/systems/Overview.do" page
+     And I wait until I see the name of "ssh-minion", refreshing the page
+     And I wait until onboarding is completed for "ssh-minion"
+
+  Scenario: Check contact method of this salt-ssh system
+    Given I am on the Systems overview page of this "ssh-minion"
+    Then I should see a "Push via SSH" text
+
+  Scenario: Run a remote command on XMLRPC-bootstrapped ssh-minion
+    Given I am authorized as "testing" with password "testing"
+    And I follow "Salt"
+    And I follow "Remote Commands"
+    And I should see a "Remote Commands" text
+    Then I enter command "ls -lha /etc"
+    And I click on preview
+    Then I should see "ssh-minion" hostname
+    And I click on run
+    Then I wait for "3" seconds
```

Comment:
we should not introduce any wait/sleeps with magic numbers when we can.

The click on run has already a dinamic timeout
https://github.com/SUSE/spacewalk-testsuite-base/blob/92dafacbae206fda75b7dac239b8dcfeaf4d97e7/features/step_definitions/salt_steps.rb#L374

Here we can just remove it